lib/matplotlib/axes/_axes.py

```diff
@@ -7488,7 +7488,8 @@ def cohere(self, x, y, NFFT=256, Fs=2, Fc=0, detrend=mlab.detrend_none,
         """
         cxy, freqs = mlab.cohere(x=x, y=y, NFFT=NFFT, Fs=Fs, detrend=detrend,
                                  window=window, noverlap=noverlap,
-                                 scale_by_freq=scale_by_freq)
+                                 scale_by_freq=scale_by_freq, sides=sides,
```

Comment:
My code analysis tool said that `sides` and `pad_to` was unused. I assume that they should be passed on (as otherwise these will not have any effect when calling `cohere` on an `Axes`).

Reply:
Yes, that should be passed on. Seems it never worked, but nobody ever complained... You can correct this.

Reply:
They are passed on.